source/sdk/structs/ArrayList.ooc

```diff
@@ -75,25 +75,6 @@ ArrayList: class <T> extends List<T> {
 		return data[index]
 	}
 
-	indexOf: override func (element: T) -> SSizeT {
-		index := 0
-		while (index < _size) {
-			//if (memcmp(data + index * T size, element, T size) == 0) return index
-			if (this as List equals(this[index], element)) return index
-			index += 1
-		}
-		return -1
-	}
-
-	lastIndexOf: override func (element: T) -> SSizeT {
-		index := _size
-		while (index > -1) {
-			if (memcmp(data + index * T size, element, T size) == 0) return index
```

Comment:
What a broken function, they start from incorrect index and check for a condition that can never be true [in `while`, -1 is converted to a very big unsigned number] :)